heron/ui/src/python/handlers/api/metrics.py

```diff
@@ -18,7 +18,7 @@
 import json
 
 from heron.ui.src.python.handlers import base
-from heron.ui.src.python.handlers import access
+from heron.common.src.python.handler import access
```

Comment:
It is because not only `heron/ui` depends on `handler`, `heron/explorer` will also depend on `handler`. We think it would be better if we move `handler` to `heron/common`. It might be confusing that `heron/explorer` depending on part of the code in `heron/ui`.